pages/Documents/ConversationalAI/ConversationBuilder/Integrations/liveperson-functions-integrations.md

```diff
@@ -24,14 +26,14 @@ Enabling the use of LivePerson's Function (FaaS) integrations requires a configu
-    <img class="fancyimage" style="width:500px" src="img/ConvoBuilder/integrations_faaSFunctionList.png">
-    The Function list provides an easy pass-through to the Functions UI in case you still need to create or modify the relevant function. Clicking **Create New Function** or <img style="width:20px" src="img/ConvoBuilder/icon_pencilModify.png"> (pencil icon) beside a function name opens a new browser window and prompt you to log into the Functions UI where you can do the work. Once done, return here and click <img style="width:25px" src="img/ConvoBuilder/icon_functionReload.png"> (reload icon) to refresh the Function list if needed.
+    <img class="fancyimage" style="width:500px" src="img/ConvoBuilder/integrations_faaSFunctionList.png" alt="Selecting a function">
+    The Function list provides an easy pass-through to the Functions UI in case you still need to create or modify the relevant function. Clicking **Create New Function** or <img style="width:20px" src="img/ConvoBuilder/icon_pencilModify.png" alt="Pencil icon"> (pencil icon) beside a function name opens a new browser window and prompt you to log into the Functions UI where you can do the work. Once done, return here and click <img style="width:25px" src="img/ConvoBuilder/icon_functionReload.png" alt="Reload icon"> (reload icon) to refresh the Function list if needed.
     - **Function Headers**: Add the necessary data in key/value pairs to pass into the request via the header.
     - **Function Payload**: Enter the payload to pass into the function.
     - **Transform Result Script**: If applicable, use this section to write JavaScript code that transforms the raw result (typically in JSON format), so you can use the information in the bot's dialogs. For more on this, see [Transform an API result](conversation-builder-integrations-integration-basics.html#transform-an-api-result).
-    - **Custom Data Fields**: Add the fields that will store the result data in key/value pairs. Users who are tasked with creating bots can use and display this data in interactions by referencing these fields. For more on this, see [here](conversation-builder-integrations-integration-basics.html#process-api-results-with-custom-data-fields).
+    - **Custom Data Fields**: Add [the fields](conversation-builder-integrations-integration-basics.html#process-api-results-with-custom-data-fields) that will store the result data in key/value pairs. Users who are tasked with creating bots can use and display this data in interactions by referencing these fields.
 3. Click **Save**.
 
 ### Example guide
 
-For a step-by-step, example guide that uses this integration, see [here](tutorials-guides-advanced-integrations-using-liveperson-functions-with-a-bot.html).
+See [this step-by-step, example guide](tutorials-guides-advanced-integrations-using-liveperson-functions-with-a-bot.html) that uses this integration.
```

Comment:
Is the comma needed here—or say “step-by-step example guide,” or even just “step-by-step guide”?